continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/.gitignore

```diff
@@ -0,0 +1,47 @@
+release/
+target/
+**/target/
+out/
+lib
+**/lib/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+**/.flattened-pom.xml
+.flattened-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+.m2/
+
+# Avoid ignoring Maven wrapper jar file (.jar files are usually ignored)
+!/.mvn/wrapper/maven-wrapper.jar
+
+# Eclipse Files # 
```

Comment:
I don't see these files anywhere in the project structure. Do we need this? We're assuming Eclipse is the IDE of choice.

Reply:
It's a general git ignore that specs out common files to ignore. If an empty repo is created in Github, it can be created with a default `.gitignore` by selecting an option at repo create time. I don't think the settings here deviate from the default, but if they do they should. Any developer who is skilled enough to know about the ignore file will be able to customize it. This is nothing outside the box of generic, standard settings. (I won't close this until I look at it more thoroughly.)

Reply:
That's a great point - wasn't aware that this was customized to support multiple IDEs or following GitHub defaults, so that's pretty great.